programs/dynamic-bonding-curve/src/instructions/ix_swap.rs

```diff
@@ -170,14 +189,37 @@ pub fn handle_swap(ctx: Context<SwapCtx>, params: SwapParameters) -> Result<()>
         current_timestamp,
     )?;
 
+    emit_cpi!(EvtSwap {
+        pool: ctx.accounts.pool.key(),
+        config: ctx.accounts.config.key(),
+        trade_direction: trade_direction.into(),
+        has_referral,
+        params: swap_in_parameters,
+        swap_result,
+        amount_in: user_pay_input_amount,
+        current_timestamp,
+    });
+
+    emit_cpi!(EvtSwap2 {
```

Comment:
I think we can push the event at the bottom, so the indexes of other event in old swap are still the same